mllib/src/test/scala/org/apache/spark/mllib/linalg/VectorsSuite.scala

```diff
@@ -295,7 +295,9 @@ class VectorsSuite extends SparkFunSuite with Logging {
       val denseVector1 = Vectors.dense(sparseVector1.toArray)
       val denseVector2 = Vectors.dense(sparseVector2.toArray)
 
-      val squaredDist = breezeSquaredDistance(sparseVector1.asBreeze, sparseVector2.asBreeze)
+      val squaredDist = sparseVector1.toArray.zip(sparseVector2.toArray).map {
```

Comment:
There was a weird compile error on breeze's squaredDistance, so I just wrote it out here instead